### PR TITLE
Update: Added a new Homepage

### PR DESCRIPTION
**Pull request by HeyNitin**
Used quiz app's styles and changed its homepage to the category page. added a new homepage

image

image

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,11 +1,14 @@
-import "./App.css";
+import "./styles/styles.css";
+import Header from "./components/Header";
 import Routes from "./components/Routes.js";
-
+import Footer from "./components/Footer";
 function App() {
   console.log(Routes);
```
Comment (bajajcodes) [inline src/App.js:6]: Remove console log statements.

File: src/components/Footer.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+import {
+  faGithub,
+  faTwitter,
+  faLinkedin,
+} from "@fortawesome/free-brands-svg-icons";
+
+const Footer = () => {
+  return (
+    <footer className="footer">
+      <div className="social">
```
Comment (bajajcodes) [inline src/components/Footer.js:11]: These lines of  code can be programmed using array and map function.

File: src/pages/Home.js [hunk 3]
```diff
@@ -1,5 +1,26 @@
...
+          <span className="heading-sm">Ready to play?</span>Let's get Started
+        </p>
+        <button onClick={() => Navigate("/categories")} className=" btn-cta">
+          Let's Go
+        </button>
+      </div>
+      <div className="img-container">
+        <img
+          className="img-responsive"
+          src="https://i.postimg.cc/vZj3v65z/quiz.png"
```
Comment (bajajcodes) [inline src/pages/Home.js:18]: You can create a js file for storing and getting these types of static data.

File: src/pages/Categories.js [hunk 4]
```diff
@@ -1,3 +1,41 @@
...
+  return (
+    <div>
+      <div className="heading-md text-centered"> Quiz Categories</div>
+      <div className="quiz-categories">
+        {categories.map((item) => {
+          return (
+            <div className="card card-basicHeader">
+              <img className="img-responsive" src={item.img} alt="quiz-img" />
+              <div className="header">
+                <a href="pages/bollywood-quizes.html" className="heading-sub">
```
Comment (bajajcodes) [inline src/pages/Categories.js:29]: Use Link here from react-router

File: src/pages/Categories.js [hunk 5]
```diff
@@ -1,3 +1,41 @@
...
+  const [categories, setCategories] = useState([]);
+
+  useState(
+    () =>
+      (async () => {
+        try {
+          const { data } = await axios.get("/api/categories");
+          setCategories(data.categories);
+        } catch (error) {
+          console.error(error);
```
Comment (bajajcodes) [inline src/pages/Categories.js:14]: Suggestion: Handler Error, no action is been taken here if any error comes.

**Review by bajajcodes — COMMENTED** [review]
- Prefer semantic html over anything.
- Left few comments, post consideration code is good to merge.

File: src/pages/Categories.js (hunk 4 above)
Comment (HeyNitin) [reply src/pages/Categories.js:29]: Done

File: src/App.js (hunk 1 above)
Comment (HeyNitin) [reply src/App.js:6]: Done

File: src/pages/Home.js (hunk 3 above)
Comment (HeyNitin) [reply src/pages/Home.js:18]: Will do

**Pull request merged**